.gitignore

```diff
@@ -59,4 +59,7 @@ typings/
 .env
 
 # Serverless directories
-.serverless
\ No newline at end of file
+.serverless
+
+# Virtual Studio Code
```

Comment:
Laconia actually has workspace specific settings committed to the repository. Instead of ignoring the change, are there changes that you'd like to share with the community?

Reply:
I had to manually add 2 spaces as indentation the config. The config is empty otherwise, so I was wondering, because there was no config so far.

Reply:
If you take the latest from master, you would see this settings: cec7a293